build/generate-docs.ts

```diff
@@ -448,9 +448,9 @@ function createExpressionsContent() {
             }
             content += `\n### ${key}\n`;
             content += `${value.doc}\n`;
-            if ('example' in value) {
-                content += `\n\`\`\`json\n"some-property": ${JSON.stringify(value.example)}\n\`\`\`\n`;
-            }
+            value.example.syntax.method.unshift(`"${key}"`);
+            content += `\nSyntax:\n\`\`\`js\n[${value.example.syntax.method.join(', ')}]: ${value.example.syntax.result}\n\`\`\`\n`;
```

Comment:
Consider making a little helper function that wraps a string in backticks, I feel this would make this a bit more readable.

Reply:
Done.